debian/control

```diff
@@ -0,0 +1,20 @@
+Source: openstreetmap-cgimap
+Section: libs
+Priority: optional
+Maintainer: Matt Amos <zerebubuth@gmail.com>
+Build-Depends: debhelper (>=10), pkg-config, zlib1g-dev, libbrotli-dev, libxml2-dev, libpqxx-dev, libfcgi-dev, libboost-program-options-dev, libfmt-dev, libmemcached-dev, libcrypto++-dev, libargon2-dev, libyajl-dev, postgresql-all, postgresql-common (>= 148~), postgresql-server-dev-all, cmake
```

Comment:
```suggestion
Build-Depends: debhelper (>=10), pkgconf, zlib1g-dev, libbrotli-dev, libxml2-dev, libpqxx-dev, libfcgi-dev, libboost-program-options-dev, libfmt-dev, libmemcached-dev, libcrypto++-dev, libargon2-dev, libyajl-dev, postgresql-all, postgresql-common (>= 148~), postgresql-server-dev-all, cmake
```

Reply:
Build was completing successfully even without this package. Checking the build logs revealed that both pkg-config and pkgconf are automatically installed anyway.  I have removed it now from the Build-Depends list.